starters/features/leaflet-map/src/routes/basic-map/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { component$, useStyles$, useSignal } from "@builder.io/qwik";
+
+// Need import from node_modules to add correct styles by default to show map
+// change path depending our level component respect node_modules
+// If you use PNPM to install
+// import leafletStyles from './../../../node_modules/.pnpm/leaflet@1.9.4/node_modules/leaflet/dist/leaflet.css?inline';
+// If you use NPM or Yarn
+import leafletStyles from "./../../../node_modules/leaflet/dist/leaflet.css?inline";
```

Comment:
This import should be correct from the end user prospective and not for this specific file

Reply:
How do you suggest I put it? The truth is that with Qwik I have tried adding the link to the CDN globally but it hasn't gone well and I haven't been able to get it to work, maybe (surely) you can help me with it.

I am attentive to all suggestions to add them and improve my PR.

Thank you very much and I am sorry for any inconvenience that may arise. Greetings